src/App.js

```diff
@@ -0,0 +1,28 @@
+import NavBar from "./components/NavBar";
+import About from "./components/About";
+import Card from "./components/Card";
+import "./styles/common.css";
+import Arr from "./cardData";
+
+function App() {
+  const cardArr = Arr.map((ind) => (
+    <Card
+      cardImg={ind.cardImg}
```

Comment:
while creating an array of components, always define `key`, it helps react in rerendering components, read about it. Also you can also refactor this code as:

```
const App = ()  => {
  const cards = Arr.map((card) => (
    <Card key={card.id} {...card} />
  ));

  return (
    <>
      <NavBar />
      <About />
      <div className="card-grid">{cards}</div>
    </>
  );
}

export default App;
```